tools/src/main/java/org/apache/kafka/tools/VerifiableProducer.java

```diff
@@ -290,7 +290,7 @@ record = new ProducerRecord<>(topic, key, value);
         }
     }
 
-    /** Returns a string to publish: ether 'valuePrefix'.'val' or 'val' **/
+    /* Returns a string to publish: ether 'valuePrefix'.'val' or 'val' */
```

Comment:
This is javadoc, right? I mean it's really vague but it documents the method below, so I think it misses a *
```suggestion
    /** Returns a string to publish: ether 'valuePrefix'.'val' or 'val' */
```

Reply:
Not sure -- `VerifiableProducer` does not seem to be a class that users would instantiate? But happy to keep it as JavaDoc (the single line formatting indicated it's not a JavaDoc 🤷 )